components/interview/Prompts/Prompts.tsx

```diff
@@ -15,7 +15,7 @@ export default function Prompts({
   currentPromptId: string;
 }) {
   const currentIndex = prompts.findIndex(({ id }) => id === currentPromptId);
-  const t = useTranslations();
+  const t = useTranslations(`Stages.${currentIndex + 1}.Prompts`);
```

Comment:
This looks like a bug, maybe? You're indexing into the `Stages` object with a prompt ID. 

However this brings up a design issue. `Prompts` doesn't need to be nested within `Stages`, because prompt IDs are unique. Nesting them in this way will just mean that we need to pass the stage index down through the component tree. Conceptually, the translated strings object is just a lookup dictionary for unique values. Things like "Prompts" and "Stages" are just useful for us to organise those strings, and make sure that they are meaningful when we are implementing the translations.

I think my idea with using indexing into the stage was for scenarios where we have a property that is shared across stages but that doesn't have a unique identifier other than the stage index. For example something like `title`. Every stage has one, but we need the stage index to know which one to load.

(With that said, stages _do_ have an `id` property. We should probably use that, rather than an array index, because we need to support multiple 'streams' through the interview.)

Back to this, I think for prompts, an appropriate key would be something like:

```
const t = useTranslations(`Protocol.Prompts.${promptId}`);
```

Let's apply this logic to stage level attributes, too:

```
const t = useTranslations(`Protocol.Stages.${stageId}.Title`);
```

Reply:
Separately, could you make the props for `Prompts` extent the HTML Div element, and then remove the `id` prop and just spread everything over than `currentIndex` and `prompts` over the top level div? This was my bad implementation from before.